ntc_templates/templates/cisco_ios_show_platform.textfsm

```diff
@@ -0,0 +1,12 @@
+Value INDEX (\d+)
+Value PORTS (\d+)
+Value MODEL (\S+)
+Value SERIAL (\S+)
+Value MAC (\S+)
+Value HW_VER (\S+)
+Value SW_VER (\d+\.\d+\.\d+)
+
+Start
+  ^Switch.*
```

Comment:
```suggestion
  ^Switch\s+Ports\s+Model\s+Serial\s+No\.\s+MAC\s+address\s+Hw\s+Ver\.\s+Sw\s+Ver\.\s*$$
```
We want to capture the full table header to make sure the columns are what we expect and are in the order we expect them to be in. This helps identify changes between models and versions.